examples/text_labels_and_annotations/mathtext_fontset_example.py

```diff
@@ -0,0 +1,77 @@
+"""
+========================
+Mathtext Fontset Example
+========================
+
+"""
+
+import matplotlib.pyplot as plt
+
+plt.figure(figsize=(20, 10))
+
+"""A simple plot for the background"""
```

Comment:
We don't use strings as comments; please make this and the following real comments.